### ISSUE TITLE
[Bug]: grant ownership to database * to role error

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#5253]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

grant ownership to database *.* to role
![image](https://user-images.githubusercontent.com/113406637/192221400-a4b99de1-2371-4c3f-b245-3cd6d903ccce.png)

**Comment by daviszhen** [comment]
it was fixed at #5411

**Comment by heni02** [comment]
confirm,closed
